tests/common/aws_test_runner.c

```diff
@@ -218,14 +179,6 @@ static void RunTestSuites( void )
         RUN_TEST_GROUP( Serializer_Unit_JSON_deserialize );
```

Comment:
is Serializer_Unit_JSON_deserialize test still needed?

Reply:
No, but I haven't finished updating this PR to remove the serializer library

Reply:
OK. Let's remove it when you remove the serializer library.